packages/engine/src/simulation/package/state/packages/behavior_execution/config.rs

```diff
@@ -1,21 +1,19 @@
-use std::{collections::HashMap, ops::Deref};
+use std::collections::HashMap;
 
 use serde::{Deserialize, Serialize};
 
 use super::{fields::behavior::BehaviorMap, Error, Result};
 use crate::Language;
 
-// TODO: Package's experiment init message should have payload with
-//       Vec of behavior descriptions in `behavior_descs`.
 #[derive(Serialize, Deserialize)]
 pub struct BehaviorDescription {
     pub id: BehaviorId,
     pub name: String,
     pub short_names: Vec<String>,
     pub source: String,
     pub required_field_keys: Vec<String>,
+    /// serde serialized to "Python", "JavaScript" or "Rust"
```

Comment:
I don't think we should really have this comment at all? If someone needs to see how the struct is serialized they should go to the impl of `Language`

Reply:
I think this got merged in accidentally

Reply:
Removed